app/views/admin/insurances/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Edit Insurance</h1>
+<%= form_with(model: [:admin, @insurance]) do |form| %>
+    <div class="field">
+      <%= form.label :name %>
+      <%= form.text_field :name %>
+    </div>
+  
+    <div class="field">
+      <%= form.label :price_cents %>
+      <%= form.number_field :price_cents %>
+    </div>
+  
+    <div class="actions">
+      <%= form.submit %>
+    </div>
+  <% end %>
```

Comment:
It's possible to remove the duplication between `new.html.erb` and `edit.html.erb` by using a partial - https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials